### ISSUE TITLE
VolumeDriver.Mount: exit status 1

### ISSUE THREAD

**Issue by swaroopkundeti** [sapk/docker-volume-gluster#33]
- Plugin version (or commit ref) : Latest
- Docker version : 17.12.0-ce
- Plugin type : Glusterfs Plugin
- Operating system: Ubuntu 16.04.3 LTS

## Description
<pre>
docker volume inspect logs
[
    {
        "CreatedAt": "0001-01-01T00:00:00Z",
        "Driver": "sapk/plugin-gluster:latest",
        "Labels": {},
        "Mountpoint": "/var/lib/docker-volumes/gluster/logs",
        "Name": "logs",
        "Options": {
            "voluri": "node1:dev_logs"
        },
        "Scope": "local",
        "Status": {
            "TODO": "List"
        }
    }
]
<pre>
 docker run -v logs:/mnt -it ubuntu
docker: Error response from daemon: VolumeDriver.Mount: exit status 1.
...

## Logs

an 10 18:34:09 node1 dockerd[1076]: time="2018-01-10T18:34:09.589431701Z" level=error msg="fatal task error" error="starting container failed: error while mounting volume '/var/lib/docker/plugins/4bc1d7f4f641c2479f60f850f5b0fbaa3b293e67bb65e0c7c4ba5c66d8780ecc/rootfs': VolumeDriver.Mount: exit status 1" module=node/agent/taskmanager node.id=pbjcny9n4k1h2dlestuscupmh service.id=iq4n61hgp4djn3b8nfau0u330 task.id=feiuf2eevzvar0l84hvw50zph
Jan 10 18:34:09 node1 dockerd[1076]: time="2018-01-10T18:34:09Z" level=error msg="2018/01/10 18:34:09 Entering go-plugins-helpers capabilitiesPath" plugin=4bc1d7f4f641c2479f60f850f5b0fbaa3b293e67bb65e0c7c4ba5c66d8780ecc
Jan 10 18:34:09 node1 dockerd[1076]: time="2018-01-10T18:34:09Z" level=error msg="2018/01/10 18:34:09 Entering go-plugins-helpers capabilitiesPath" plugin=4bc1d7f4f641c2479f60f850f5b0fbaa3b293e67bb65e0c7c4ba5c66d8780ecc

**Comment by swaroopkundeti** [comment]
Is this plugin compatible with docker 17.12 version?

**Comment by sapk** [comment]
Yes it should. Are you sure that `node1` (the gluster server name) is resolvable by the host ?
